File: bank-service/src/main/java/rs/edu/raf/repository/CreditTypeRepository.java
package rs.edu.raf.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import rs.edu.raf.model.credit.CreditType;

@Repository
public interface CreditTypeRepository extends JpaRepository<CreditType, Long> {
}

File: bank-service-kotlin/src/main/kotlin/org/example/bankservicekotlin/repository/CreditTypeRepository.kt
package org.example.bankservicekotlin.repository

import org.example.bankservicekotlin.model.credit.CreditType
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface CreditTypeRepository:JpaRepository<CreditType, Long> {

}